### ISSUE TITLE
Unity Log errors with Valheim+

### ISSUE THREAD

**Issue by dyoung522** [nxPublic/ValheimPlus#11]
Getting errors in log:

```

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ValheimPlus.ValheimMultipliersPlugin+ModifyPlacingRestrictionOfGhost.Prefix (Player& __instance, System.Int32& ___m_placementStatus, UnityEngine.GameObject& ___m_placementGhost, UnityEngine.GameObject& ___m_placementMarkerInstance, System.Int32& ___m_placeRotation) (at <b4defa7c27074b7b9df63cfc4d21a06e>:0)
(wrapper dynamic-method) Player.DMD<Player::UpdatePlacementGhost>(Player,bool)
Player.LateUpdate () (at <f24ccd8702d04f38afc5201e71155009>:0)
```

**Comment by dyoung522** [comment]
Got this error tonight while trying Valheim+

Otherwise, awesome mod! THANK YOU!

**Comment by nxPublic** [comment]
Issue resolved in 0.6
